plugins/powerd/fu-plugin-powerd.c

```diff
@@ -31,28 +31,21 @@ fu_plugin_destroy (FuPlugin *plugin)
 gboolean
 fu_plugin_startup (FuPlugin *plugin, GError **error)
 {
-	FuPluginData *data = fu_plugin_get_data (plugin);
+	FuPluginData     *data = fu_plugin_get_data (plugin);
```

Comment:
is that right? I thought we only block aligned in structs?

Reply:
AlignConsecutiveDeclarations is global in the sense that affects structs and blocks of code the same

Reply:
There's no way of making it non-global?

Reply:
not that I can find on https://clang.llvm.org/docs/ClangFormatStyleOptions.html :(

Reply:
maybe AlignConsecutiveDeclarations=false makes more sense then perhaps?

Reply:
done